src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -115,6 +128,17 @@ public static void showFirstPersonOnly(Model model) {
     }
 
     /**
+     * Updates {@code model}'s filtered list to show only the first person in the {@code model}'s address book.
+     */
+    public static void showFirstBlacklistedPersonOnly(Model model) {
+        ReadOnlyPerson person = model.getAddressBook().getBlacklistedPersonList().get(0);
+        final String[] splitName = person.getName().fullName.split("\\s+");
```

Comment:
Perhaps the magic constant `\\s+` can be refactored out to improve code readability.